src/pages/CatIndex.js

```diff
@@ -1,10 +1,40 @@
 import React, { Component } from 'react'
+import { Card, CardTitle, Col } from 'reactstrap'
+import { NavLink } from 'react-router-dom'
 
 class CatIndex extends Component {
-  render () {
-    return (
-      <div>Footer</div>
+  constructor(props){
+    super(props)
+    }
+      render () {
+        let {cat}= this.props
+          return (
+      <>
+        <h3>Meet the Cats!</h3>
+        <br />
+        <Col sm="6">
+          {this.props.cats && this.props.cats.map(cat => {
```

Comment:
Since you are destructuring cats out of props, your line 16 can be `{cats && cats.map(....`

Reply:
Hi sarah I changed it to cat && cat.map and it ran but cats didn't, did you mean that or cats**?